pkgs/development/tools/analysis/lcov/default.nix

```
@@ -1,24 +1,16 @@
- {stdenv, fetchurl, fetchpatch, perl, perlPackages, makeWrapper }:
+ {stdenv, fetchFromGitHub, perl, perlPackages, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  name = "lcov-1.14";
-
-  src = fetchurl {
-    url = "mirror://sourceforge/ltp/${name}.tar.gz";
-    sha256 = "06h7ixyznf6vz1qvksjgy5f3q2nw9akf6zx59npf0h3l32cmd68l";
+  version = "v1.15";
+  name = "lcov-${version}";
```

```suggestion
  pname = "lcov";
  version = "v1.15";
```

You are right. We need to define pname then.

---

Let's rather set `version = "1.15";` and then `rev = "v${version}"`.

---

Pname is fine. Are we trying to transition from name to pname? A quick `grep | wc` a week or two ago have me roughly 50/50 between the two.